src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,72 @@
 package core.basesyntax;
 
+import java.io.*;
+import java.util.Arrays;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        try {
+            file.createNewFile();
+            BufferedReader reader = new BufferedReader(new FileReader(file));
+            int value = reader.read();
+            while (value != -1) {
+                stringBuilder.append((char) value);
+                value = reader.read();
+            }
+
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException(e);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        String str = stringBuilder.toString();
+        System.out.println(str);
+        String[] lines = str.split("\n");
+        System.out.println(Arrays.toString(lines));
+        System.out.println(lines.length);
+        String[] numbers = str.trim().split("\\D+");
+        Integer[] num = new Integer[numbers.length - 1];
+        System.out.println(num.length);
+        for (int i = 0; i < num.length; i++) {
+            num[i] = Integer.valueOf(numbers[i+1]);
```

Comment:
Using `Integer.valueOf(numbers[i+1])` assumes that all elements in `numbers` after the first are valid integers. This can throw a `NumberFormatException` if any element is not a valid integer.